addons/mass_mailing/migrations/13.0.2.0/pre-migration.py

```diff
@@ -0,0 +1,61 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_model_renames = [
+    ('mail.mail.statistics', 'mailing.trace'),
+    ('mail.mass_mailing', 'mailing.mailing'),
+    ('mail.mass_mailing.contact', 'mailing.contact'),
+    ('mail.mass_mailing.list', 'mailing.list'),
+    ('mail.mass_mailing.list_contact_rel', 'mailing.contact.subscription'),
+]
+
+_table_renames = [
+    ('mail_mail_statistics', 'mailing_trace'),
+    ('mail_mass_mailing', 'mailing_mailing'),
+    ('mail_mass_mailing_contact', 'mailing_contact'),
+    ('mail_mass_mailing_list', 'mailing_list'),
+    # many2many tables
+    ('mail_mass_mailing_list_contact_rel', 'mailing_contact_list_rel'),
```

Comment:
table mail_mass_mailing_list_contact_rel does not exist so remove it

Reply:
in previous mail.mass_mailing.contact an current mailing.contact model there is a many2many model contact_ids field that use this relation table and it's changed

Reply:
No. You misstake and write mail_mass_mailing_list_contact_rel instead of mail_mass_mailing_contact_list_rel.
But you write to rename mail_mass_mailing_contact_list_rel later. So you need just remove this line